mrmustard/__init__.py

```diff
@@ -46,6 +50,22 @@ def __init__(self):
         self.HOMODYNE_SQUEEZING = 10.0
         # misc
         self.PROGRESSBAR = True
+        self._seed = np.random.randint(0, 2**32)
+        self.random_state = RandomState(MT19937(SeedSequence(self._seed)))
```

Comment:
For the sake of clarity, I believe this should be name `rng` or `random_number_generator` as it provides random numbers and not the state of the generator.